<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeManagerInfoDialog</class>
 <widget class="QDialog" name="ChangeManagerInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="confirmPwLineEdit"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="oldPwLineEdit"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="newPwLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>确认新密码：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>输入旧密码：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>输入新密码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oldPwLineEdit</tabstop>
  <tabstop>newPwLineEdit</tabstop>
  <tabstop>confirmPwLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeManagerInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeManagerInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
